apitools/base/py/credentials_lib.py

```diff
@@ -39,9 +39,19 @@
 #
 # pylint: disable=wrong-import-order,ungrouped-imports
 try:
-    from oauth2client import gce, locked_file, multistore_file
+    from oauth2client.contrib import gce
```

Comment:
Can gce, locked_file and multistore_file come from old and new locations at same time?